dotnet/src/Extensions/Planning.StepwisePlanner/EmbeddedResource.cs

```diff
@@ -12,10 +13,10 @@ internal static class EmbeddedResource
     internal static string Read(string name)
     {
         var assembly = typeof(EmbeddedResource).GetTypeInfo().Assembly;
-        if (assembly == null) { throw new PlanningException(PlanningException.ErrorCodes.InvalidConfiguration, $"[{s_namespace}] {name} assembly not found"); }
+        if (assembly == null) { throw new SKException($"[{s_namespace}] {name} assembly not found"); }
```

Comment:
Most likely it won't be null. We have similar logic in a few other places, so it would make sense to change it in one go rather than scatter over different not related PRs. I'll change the null check style here, but we better to do it all at once if we think we want to use it. Feel free to create issues to track those. I've added an item to my TODO list to remove this the assembly == null check in SK code so I can do it later in a separate PR to keep this PR up to the point.